src/dashboard/apps/web/src/components/auth/cyberpunk.css

```
@@ -0,0 +1,327 @@
+  background: linear-gradient(135deg, #ff9500, #ffd700, #00f0ff);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+
+/* Animation: Fade in up */
+.animate-fade-in-up {
+  animation: fadeInUp 0.5s ease-out forwards;
+}
+
+@keyframes fadeInUp {
```

Stylelint (17.11.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)